src/main/java/com/woowacourse/matzip/application/response/RestaurantResponse.java

```diff
@@ -0,0 +1,39 @@
+package com.woowacourse.matzip.application.response;
+
+import com.woowacourse.matzip.domain.restaurant.Restaurant;
+import lombok.Getter;
+
+@Getter
+public class RestaurantResponse {
+
+    private Long id;
+    private Long categoryId;
+    private Long campusId;
+    private String name;
+    private String address;
+    private Long distance;
+    private String kakaoMapUrl;
+    private String imageUrl;
+
+    private RestaurantResponse() {
+    }
+
+    private RestaurantResponse(final Long id, final Long categoryId, final Long campusId, final String name,
+                               final String address, final Long distance, final String kakaoMapUrl,
+                               final String imageUrl) {
+        this.id = id;
+        this.categoryId = categoryId;
+        this.campusId = campusId;
+        this.name = name;
+        this.address = address;
+        this.distance = distance;
+        this.kakaoMapUrl = kakaoMapUrl;
+        this.imageUrl = imageUrl;
+    }
+
+    public static RestaurantResponse from(final Restaurant restaurant) {
+        return new RestaurantResponse(restaurant.getId(), restaurant.getCategoryId(), restaurant.getCampusId(),
+                restaurant.getName(), restaurant.getAddress(), restaurant.getDistance(), restaurant.getKakaoMapUrl(),
+                restaurant.getImageUrl());
```

Comment:
이건 진짜(진짜로) 사소한 스타일인데요!

생성자에서 이렇게 길게 나온다면 오히려 세로로 길게 늘려두었을때 가독성이 더 올라간다라고 생각하는 편입니다! 적용안하셔도돼요 ㅎㅎ
```suggestion
        return new RestaurantResponse(
                restaurant.getId(), 
                restaurant.getCategoryId(), 
                restaurant.getCampusId(),
                restaurant.getName(), 
                restaurant.getAddress(),
                restaurant.getDistance(), 
                restaurant.getKakaoMapUrl(),
                restaurant.getImageUrl()
        );
```